Makefile

```diff
@@ -4,6 +4,15 @@ all: lkmpg.tex
 	bibtex main >/dev/null || echo
 	pdflatex -shell-escape $< 2>/dev/null >/dev/null
 
+html: lkmpg.tex
+	make4ht --shell-escape --utf8 --format html5 --config html.cfg --output-dir html lkmpg.tex
+	ln -sf ./html/lkmpg.html ./html/index.html
+	rm -f *.xref *.svg *.tmp *.html *.css *.4ct *.4tc *.dvi *.lg *.idv
+
 clean:
 	rm -f *.dvi *.aux *.log *.ps *.pdf *.out lkmpg.bbl lkmpg.blg lkmpg.lof lkmpg.toc
 	rm -rf _minted-lkmpg
+	rm -f *.xref *.svg *.tmp *.html *.css *.4ct *.4tc *.dvi *.lg *.idv
```

Comment:
Avoid unintended deletions.

Reply:
Did you mean `rm` command in html target?
Or maybe some specific file format?